nixos/tests/hedgedoc.nix

```diff
@@ -45,20 +38,21 @@ import ./make-test-python.nix ({ pkgs, lib, ... }:
 
     hedgedocPostgresWithUNIXSocket = { ... }: {
       systemd.services.hedgedoc.after = [ "postgresql.service" ];
+      # In a non-test scenario, this should be emplaced by some
+      # secrets handling mechanism like sops-nix or colmena secrets.
+      systemd.tmpfiles.rules = [
+        "L+ /run/secrets/hedgedoc-db-password - - - - ${pkgs.writeText "password" "snakeoilpassword"}"
+      ];
       services = {
         hedgedoc = {
           enable = true;
           settings.db = {
             dialect = "postgres";
             user = "hedgedoc";
-            password = "$DB_PASSWORD";
+            password = lib.stringFromRuntimeFile "/run/secrets/hedgedoc-db-password";
```

Comment:
This is not a great example as we can just set the CMD_DB_PASSWORD env and avoid this all together.